src/main/java/org/sagebionetworks/web/client/jsinterop/ReactDOM.java

```diff
@@ -17,4 +17,8 @@ public interface Callback {
   public static native ReactDOMRoot createRoot(Element container);
 
   public static native boolean unmountComponentAtNode(Element container);
+
+  public static native void flushSync();
+
+  public static native void flushSync(Callback callback);
```

Comment:
[Register this ReactDOM function](https://react.dev/reference/react-dom/flushSync)

Reply:
Could you add a comment indicating that people should probably not need to use this?
something like: "flushSync is used to force React to flush any pending work and update the DOM synchronously, and  can significantly hurt performance. Use sparingly.